patch-diff-report-tool/config/suppressions.xml

```diff
@@ -17,4 +17,7 @@
     <!-- Until https://github.com/checkstyle/checkstyle/issues/3496 and when we have time to fix this in sevntu-->
     <suppress checks="ReturnCount" files=".*CheckstyleRecord\.java|DiffReport\.java|MergedConfigurationModule\.java"/>
 
+    <!-- MessageKey annotation is used for documentation and applies only to checks -->
+    <suppress id="missingMessageKeyAnnotation" files=".*" />
```

Comment:
This seems like an issue with the logic of `missingMessageKeyAnnotation` if it thinks anything in this project is a check. None of the files I see ends with "Check".

What files are getting violations?